// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

message t_dln_traffic_stats_item {
   required uint32 count = 1;
   required uint32 total = 2;
}

message t_dln_stats {
   required uint32 du_ue_index = 1;
   required uint32 is_srb = 2;
   required uint32 rb_id = 3;
   required t_dln_traffic_stats_item sdu_new_bytes = 4;
}

message rlc_dl_north_stats {
   required uint64 timestamp = 1;
   repeated t_dln_stats stats = 2;
}





